build-clang-with-args.sh

```diff
@@ -118,6 +118,8 @@ cmake --build "${llvm_build_dir}" \
   --parallel $(( $(nproc) + 2 )) \
   --target install-distribution
 
+cd "${build_top_dir}"
```

Comment:
Everything after this `cd` seems to use absolute file paths for querying just-built information, and I wanted to be in this directory so if `ARTIFACT_STAGING_DIR` is not set, the tarball ends up in the same directory as the invoked script. Though maybe it pays to pass another argument and be more explicit.